Sources/Configuration/SentrySwiftUI.xcconfig

```diff
@@ -5,3 +5,4 @@ MACOSX_DEPLOYMENT_TARGET = 10.15
 IPHONEOS_DEPLOYMENT_TARGET = 13.0
 WATCHOS_DEPLOYMENT_TARGET = 6.0
 TVOS_DEPLOYMENT_TARGET = 13.0
+MACH_O_TYPE = mh_dylib
```

Comment:
`m`: Was SwiftUI a dynamic library before? Does this change have any implications for SwiftUI?

Reply:
It was not static before, I just move the configuration from the xcodeproj to a config file.